x-pack/legacy/plugins/graph/public/components/field_manager/field_picker.tsx

```diff
@@ -37,9 +37,9 @@ export function FieldPicker({
       // only update the field options if the popover is not open currently.
       // This is necessary because EuiSelectable assumes options don't change
       // on their own.
-      setFieldOptions(toOptions(allFields));
+      setFieldOptions(toOptions(Object.values(fieldMap)));
```

Comment:
`Object.values` is not supported in IE11. Is Graph expected to work with IE at all?

Reply:
Yes, all of Kibana should work in IE11. There are polyfills in place though for things like this.